CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 ## [0.1] - 2024-05-29
 ### Added
+- `CLI` Handler
```

Comment:
I will change it to `handler`, is that OK?

Reply:
Yes.